lib/bolt/analytics.rb

```diff
@@ -93,6 +93,9 @@ def self.load_config(filename, logger)
     def self.write_config(filename, config)
       FileUtils.mkdir_p(File.dirname(filename))
       File.write(filename, config.to_yaml)
+    rescue StandardError => e
+      Bolt::Logger.warn_once('unwriteable_file', "Could not write analytics configuration to #{filename}; "\
```

Comment:
I think "continuing" is redundant, since this is a warning.

We should also disable analytics in this case, otherwise we'll generate a new uuid every run. So maybe in place of "continuing" we could say "disabling analytics".

Reply:
It seems like the easiest way to do that is to have this warn then fail, and the failure will get caught by `build_client` and create a NoopClient and print a message.